src/pages/ItemList.js

```diff
@@ -45,67 +45,93 @@ function ItemList({ token }) {
     fetchDocs(token);
   }, [token]);
 
-  useEffect(() => {
-    // Update "now" every second to force the page to re-render and keep the checkbox state up to date;
-    const intervalId = setInterval(() => {
-      setNow(Date.now());
-    }, ONE_SECOND);
-    return () => clearInterval(intervalId);
-  }, []);
-
   const isClicked = async ({ target: { checked, id } }) => {
```

Comment:
I feel like the name `isClicked` may not be the ideal name here. Maybe we should rename it do something like `handleClick`?